src/lib/annotations/doc/DocDrawingDialog.js

```diff
@@ -1,6 +1,8 @@
 import AnnotationDialog from '../AnnotationDialog';
 
 class DocDrawingDialog extends AnnotationDialog {
+    visible = false;
```

Comment:
JSDoc in here

Reply:
JSDoc is in #364, I separated them to make it easier to read. DocDrawingDialog is simply a stub in this PR.